data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            content: "- Create business cases to boost sustainability ([Circular Economy](https://www.wbcsd.org/Programs/Circular-Economy), [Cities and Mobility](https://www.wbcsd.org/Programs/Cities-and-Mobility), [Climate & Energy](https://www.wbcsd.org/Programs/Climate-and-Energy), [Food & Nature](https://www.wbcsd.org/Programs/Food-and-Nature), [People](https://www.wbcsd.org/Programs/People) and [Redefining Value](https://www.wbcsd.org/Programs/Redefining-Value)."
+            + "\n- Run the [Global Agribusiness Alliance](https://www.wbcsd.org/Sector-Projects/Global-Agribusiness-Alliance) in which some of the worst environmental offenders slightly increase their sustainability."
+            + "\n[More projects can be found online](https://www.wbcsd.org/).",
+            updatedBy: "Alexander", updatedAt: "2020-02-16",
+        },
+        established: {
+            content: "[1995](https://www.wbcsd.org/Overview/Our-history]",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "Giving Green",
+            link: "https://www.idinsight.org/givinggreen",
+            keywords: ["ngo", "efficiency-assessment"],
+            label: "giving-green",
+        },
+        description: {
+            content: "[IDinsight](https://www.idinsight.org/) works on assessing the efficiency (ratio of effort to impact) of initiatives fighting poverty."
+            + " Giving Green is a project by IDinsight that tries to give clear funding recommendations."
+            + " The website describes the project as being in Beta stage. Many of the recommendations are drawn from [IDinsight itself](https://www.idinsight.org/giving-green-short-term-recommendations)"
+            + " or are sourced from a [report by the Founder's Pledge](https://founderspledge.com/stories/climate-change-executive-summary) and [recommend initiatives focusing on US politics or the REDD+ mechanism](https://www.idinsight.org/giving-green-long-term-recommendations)."
+            + " Giving Green attempts to solve a key question in the fight against the climate crisis but seems to be in a very early stage in doing so.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "Clean Air Taskforce",
+            link: "https://www.catf.us/",
+            keywords: ["ngo", "legislation", "energy", "technology"],
+            label: "catf",
+        },
+        description: {
+            content: "The CATF lobbies US governmental institutions to reduce emissions of greenhouse gases."
+            + " In addition they work to structure future [technical options for carbon-reduced energy production](https://www.catf.us/work/advanced-energy-systems/)."
+            + " CATF aims to assist corporates and governments in designing future energy systems based on renewables, biofuels, nuclear and systemic improvements of grids and consumption.",
```

Comment:
are they are just aiming to assist or are they actually doing it

Reply:
Probably both, I changed it to `CATF lobbies and supports`

Reply:
ok